plyer/platforms/android/flash.py

```diff
@@ -7,13 +7,20 @@
 from plyer.facades import Flash
 from jnius import autoclass
 from plyer.platforms.android import activity
-
-Camera = autoclass("android.hardware.Camera")
-CameraParameters = autoclass("android.hardware.Camera$Parameters")
-SurfaceTexture = autoclass("android.graphics.SurfaceTexture")
-PackageManager = autoclass('android.content.pm.PackageManager')
-pm = activity.getPackageManager()
-flash_available = pm.hasSystemFeature(PackageManager.FEATURE_CAMERA_FLASH)
+from plyer.platforms.android import SDK_INT
+if SDK_INT >= 23:
+  Context = autoclass('android.content.Context')
+  CameraManager = activity.getSystemService(Context.CAMERA_SERVICE)
+  CameraCharacteristics = autoclass("android.hardware.camera2.CameraCharacteristics")
+  CamId = CameraManager.getCameraIdList()[0] # First Camera
```

Comment:
Personally I don't like this method to force it to be [0]. For loop with checks would be one way of dealing with this. All Androids do not have a camera so it would be basically better to find out if it's supported.